File: Web/RMA-Web-Shell/projects/fincare/src/app/shared/models/finpayee.ts
import { DebtorStatusEnum } from '../enum/debtor-status.enum';

export class FinPayee {
    rolePlayerId: number;
    finPayeNumber: string;
    isAuthorised: boolean;
    industryId: number;
    authroisedBy: string;
    authorisedDate: Date;
    debtorStatus?: DebtorStatusEnum;
}

File: Service/Billing/RMA.Service.Billing.Contracts/Entities/FinPayee.cs

namespace RMA.Service.Billing.Contracts.Entities
{
    public class FinPayee
    {
        public int RolePlayerId { get; set; }
        public string FinPayeNumber { get; set; }
        public bool IsAuthorised { get; set; }
        public string AuthroisedBy { get; set; }
        public System.DateTime? AuthorisedDate { get; set; }
        public int? IndustryId { get; set; } // IndustryId
    }
}